src/bunit.web/EventDispatchExtensions/TriggerEventDispatchExtensions.cs

```diff
@@ -80,7 +80,8 @@ private static Task TriggerBubblingEventAsync(ITestRenderer renderer, IElement e
 		return Task.WhenAll(eventTasks);
 	}
 
-	private static List<Task> GetDispatchEventTasks(ITestRenderer renderer, IElement element, string eventName, EventArgs eventArgs)
+	private static List<Task> GetDispatchEventTasks(ITestRenderer renderer, IElement element, string eventName,
+	EventArgs eventArgs)
```

Comment:
```suggestion
	private static List<Task> GetDispatchEventTasks(ITestRenderer renderer, IElement element, string eventName, EventArgs eventArgs)
```

That line break looks a bit weird to me.

I dont mind breaking arguments down to new lines if there are many arguments, but in that case, I want all to be on their own line, or none, e.g.:

```
	private static List<Task> GetDispatchEventTasks(
		ITestRenderer renderer,
		IElement element,
		string eventName,
		EventArgs eventArgs)
```

Reply:
Thanks for noticing that. Sometimes Rider does that when the line is longer than 120 characters (or so). And it does it sometimes in a weird way.